build.sbt

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2012 Typesafe, Inc. <http://www.typesafe.com>
+ */
+
+val incrementalVersion = "0.1.0-M1-28b0fa40021bdb755efbba033265601dee065a3d-SNAPSHOT"
```

Comment:
This version should be any sbt/incrementalcompiler with sbt/incrementalcompiler#25 and compiled with sbt-datatype with sbt/sbt-datatype#13